### PR TITLE
[DOC UPDATE] Translate Tutorial to Chinese 5/14

### PR DESCRIPTION
**Pull request by kingswanwho**
# [DOC UPDATE] Translate Tutorial to Chinese 5/14

## Description

Translate three tutorials to Chinese:

- 030-analyzing-the-yelp-academic-dataset.md
- 040-learn-drill-with-the-mapr-sandbox.md
- 005-about-the-mapr-sandbox.md

Overall 5 tutorials have been translated, 9 tutorials to go.

## Documentation

Please refer to Chinese version's tutorial.

## Testing
This is a doc update, no test here.

### CODE REVIEW

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 1]
```diff
@@ -401,12 +380,12 @@ data so you can apply even deeper SQL functionality. Here is a sample query:
...
+与 Drill 社区保持密切联系来获得更多的特性以及了解即将到来的活动。
 
-To learn more about Drill, please refer to the following resources:
+想更多了解 Drill，请参考如下资源：
 
-  * Download Drill here: <http://getdrill.org/drill/download>
-  * [10 reasons we think Drill is cool]({{site.baseurl}}/docs/why-drill)
-  * [A simple 10-minute tutorial]({{ site.baseurl }}/docs/drill-in-10-minutes>)
-  * [More tutorials]({{ site.baseurl }}/docs/tutorials-introduction/)
+  * 下载 Drill: <http://getdrill.org/drill/download>
```
Comment (luocooong) [inline _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md:387]: Replace the download link to official. Please sync to the english docs.

**Review by luocooong — COMMENTED** [review]
@kingswanwho Thanks for the work. Is glad to see you choose a high-quality human translation, not the machine translation.

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 2]
```diff
@@ -4,41 +4,33 @@ slug: "Analyzing the Yelp Academic Dataset"
...
-value from your data in just a few minutes.
-
-Let’s quickly walk through the steps required to install Drill and run it
-against the Yelp data set. The publicly available data set used for this
-example is downloadable from [Yelp](http://www.yelp.com/dataset_challenge)
-(business reviews) and is in JSON format.
+
+Apache Drill 是发展最快的开源项目之一，社区快速发展并每月保持新版本发布。Drill 的与众不同之处在于敏捷性和灵活性。
+为了满足 SQL 查询 Hadoop，并规模化减少延迟，Drill允许用户不必进行 ETL 流程或者 预先定义 schema。文件可以是任意格式，比如：纯文本，JSON 或者 Parquet。
+数据可以是简单的字符串，整数，日期，也可以是更复杂的多结构数据，比如嵌套地图和数组。数据可以保存在任意文件系统，本地或者分布式，比如 HDFS 或者 S3。Drill 具备 “no schema” 方法，
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md]: resolved

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 3]
```diff
@@ -154,17 +146,14 @@ You can directly query self-describing files such as JSON, Parquet, and text. Th
...
 
-Note that the attributes column in the Yelp business data set has a different
-element for every row, representing that businesses can have separate
-amenities. Drill makes it easy to quickly access data sets with changing
-schemas.
+请注意 Yelp 商家数据集中，属性列的每一行都有不同的元素，代表商家有不同的便利设施。Drill 通过改变 schema 更简单的快速访问数据集。
 
-First, change Drill to work in all text mode (so we can take a look at all of
-the data).
+首先，更改配置使 Drill 可以识别所有的文本格式（我们便可查看所有的数据）。
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md:156]: resolved

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 4]
```diff
@@ -310,13 +296,11 @@ of the reviews themselves.
...
 
-#### Create a view with the combined business and reviews data sets
+#### 创建了连接商户和评论数据集后的视图
 
-Note that Drill views are lightweight, and can just be created in the local
-file system. Drill in standalone mode comes with a dfs.tmp workspace, which we
-can use to create views (or you can can define your own workspaces on a local
-or distributed file system). If you want to persist the data physically
-instead of in a logical view, you can use CREATE TABLE AS syntax.
+Drill 的视角是轻量级的，且只是创建在本地文件系统。
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md]: resolved

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 5]
```diff
@@ -337,21 +321,16 @@ Let’s get the total number of records from the view.
     | 1125458    |
     |------------|
 
-In addition to these queries, you can get many deep insights using
-Drill’s [SQL functionality]({{ site.baseurl }}/docs/sql-reference). If you are not comfortable with writing queries manually, you
-can use a BI/Analytics tools such as Tableau/MicroStrategy to query raw
-files/Hive/HBase data or Drill-created views directly using Drill [ODBC/JDBC
-drivers]({{ site.baseurl }}/docs/odbc-jdbc-interfaces).
+在这些查询之外，你可以利用 Drill 的 [SQL 函数]({{ site.baseurl }}/docs/sql-reference) 进行更深入的分析。
+如果你不习惯手写查询，你可以利用商务智能分析工具如 Tableau/MicroStrategy 来查询原始 files/Hive/HBase 数据，或者通过[ODBC/JDBC
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md]: resolved

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md [hunk 6]
```diff
@@ -337,21 +321,16 @@ Let’s get the total number of records from the view.
...
+drivers]({{ site.baseurl }}/docs/odbc-jdbc-interfaces)直接创建 Drill 视图。
 
-The goal of Apache Drill is to provide the freedom and flexibility in
-exploring data in ways we have never seen before with SQL technologies. The
-community is working on more exciting features around nested data and
-supporting data with changing schemas in upcoming releases.
+Apache Drill 的目标在于通过 SQL 技术提供前所未见的方式自由和灵活的探索数据。社区正在围绕嵌套数据努力提供更多令人兴奋的特性，
```
[the remark this answers is not in the document]
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md]: resolved

File: _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md (hunk 1 above)
Comment (kingswanwho) [reply _docs/zh/tutorials/030-analyzing-the-yelp-academic-dataset.md]: resolved

**Comment by kingswanwho** [comment]
All issues @luocooong mentioned, has been resolved

**Review by luocooong — APPROVED** [review]
LGTM. +1
@dzamo It is time to merge. thanks

**Comment by dzamo** [comment]
> LGTM. +1
> @dzamo It is time to merge. thanks

Beat me to it!

**Pull request merged**